site/app/models/Config.php

```diff
@@ -546,6 +552,13 @@ public function useHomeworkLibrary(): bool {
         return $this->homework_library_params['enabled'];
     }
 
+    /**
+     * @return bool
+     */
+    public function canAccessHomeworkLibrary(): bool {
+        return $this->homework_library_params['allowed'];
+    }
```

Comment:
I think this should go in User.php like the accessFaculty() function along with the earlier thing that checks access level. Then return true if the manager is allowed and the user is a superuser. Maybe rename to something like canAccessLibraryManager to clarify that this is for the management page